documentation/book/proc-configuring-kafka-bridge-authentication-scram-sha-512.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-authentication.adoc
+
+[id='proc-configuring-kafka-bridge-authentication-scram-sha-512-{context}']
+= Configuring SCRAM-SHA-512 authentication in Kafka Bridge
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+* Username of the user which should be used for authentication
+* If they exist, the name of the `Secret` with the password used for authentication and the key under which the password is stored in the `Secret`
+
+.Procedure
+
+. (Optional) If they do not already exist, prepare a file with the password used in authentication and create the `Secret`.
```

Comment:
We should mentione here that secrets created by the UO can be used out of the box.

Reply:
Secrets created by the User Operator may be used.